src/defaults.js

```diff
@@ -23,6 +23,7 @@ Selectize.defaults = {
 	closeAfterSelect: false,
 
 	scrollDuration: 60,
+	dropdownOnBackspaceGotoTop: false,
```

Comment:
the name is a bit awkward.
Maybe something like: deselectBehavior? 
Maybe instead of a boolean we can make it a string/enum to allow for other options in the future?

Reply:
Can be done, will do, and get back to you.